tutorials/desi/picca_convert_transmission.py

```diff
@@ -2,52 +2,62 @@
 
 import argparse
 
-from picca import converters
+from picca import raw_io
```

Comment:
Out of curiosity, why is this script in "tutorials/desi/"? I'd have imagine this would be in the main "/bins/" folder.